modules/luci-mod-system/htdocs/luci-static/resources/view/system/startup.js

```diff
@@ -74,25 +74,26 @@ return view.extend({
 		var table = E('table', { 'class': 'table' }, [
 			E('tr', { 'class': 'tr table-titles' }, [
 				E('th', { 'class': 'th' }, _('Start priority')),
+				E('th', { 'class': 'th' }, _('Stop priority')),
 				E('th', { 'class': 'th' }, _('Initscript')),
 				E('th', { 'class': 'th nowrap cbi-section-actions' })
 			])
 		]);
 
 		for (var init in initList)
-			if (initList[init].index < 100)
-				list.push(Object.assign({ name: init }, initList[init]));
+			list.push(Object.assign({ name: init }, initList[init]));
 
 		list.sort(function(a, b) {
-			if (a.index != b.index)
-				return a.index - b.index
+			if (a.startlevel != b.startlevel)
+				return a.startlevel - b.startlevel
 
-			return a.name > b.name;
+			return ('' + a.name).localeCompare(b.name);
 		});
 
 		for (var i = 0; i < list.length; i++) {
 			rows.push([
-				'%02d'.format(list[i].index),
+				list[i].startlevel == '255' ? '--' : '%02d'.format(list[i].startlevel),
```

Comment:
Please do a numerical comparison here and move the success case first:
```javascript
list[i].startlevel != 255 ? '%02d'.format(list[i].startlevel) : '---'
```

Reply:
I think the condition was read+understandable better before
I've not checked what happens when someone sets in script `START='veryfirst'`